### PR TITLE
Migrate bicep example: 101/databricks-workspace

### PR DESCRIPTION
**Pull request by StephenWeatherford**
Added bicep support to this sample by integrating bicep.main from the example in https://github.com/Azure/bicep/tree/main/docs/examples/101/databricks-workspace

Bicep PR: https://github.com/Azure/bicep/pull/3152

### CODE REVIEW

**Comment by StephenWeatherford** [comment]
FYI @sunainakrishnamoorthy

**Comment by azure-quickstarts** [comment]
@mamccrea - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@mamccrea - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by StephenWeatherford** [comment]
fyi @sunainakrishnamoorthy you're welcome to review if you'd like.

File: quickstarts/microsoft.databricks/databricks-workspace/metadata.json [hunk 1]
```diff
@@ -6,5 +6,5 @@
   "summary": "Deploy an Azure Databricks workspace.",
   "githubUsername": "sunainakrishnamoorthy",
```
Comment (sunainakrishnamoorthy) [inline quickstarts/microsoft.databricks/databricks-workspace/metadata.json:7]: Please update this to your username, generally it is the last updated username and date

Comment (StephenWeatherford) [reply quickstarts/microsoft.databricks/databricks-workspace/metadata.json:7]: @bmoore-msft I always took this to mean the original author or the one who "owns" it.  I'm certainly not going to put my name on all of the converted bicep samples, so if sunainakrishnamoorthy doesn't want his there, do you have any other suggestions?

Comment (bmoore-msft) [reply quickstarts/microsoft.databricks/databricks-workspace/metadata.json:7]: the property is *not* the last updated, it's the user who submitted the sample... so you would not change them on update/migration.

This is a doc sample, so you can leave as is (we can't remove it) or see if @mamccrea wants to become the owner.

Comment (sunainakrishnamoorthy) [reply quickstarts/microsoft.databricks/databricks-workspace/metadata.json:7]: Thanks for the clarification, sorry about the confusion.

File: quickstarts/microsoft.databricks/databricks-workspace/main.bicep [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    managedResourceGroupId: managedResourceGroup.id
+    parameters: {
+      enableNoPublicIp: {
+        value: disablePublicIp
+      }
+    }
+  }
+}
+
+resource managedResourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' existing = {
```
Comment (msaranga) [inline quickstarts/microsoft.databricks/databricks-workspace/main.bicep:35]: Is this creating managed Resource group. If yes, we don't need to create managed resource group

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.databricks/databricks-workspace/main.bicep (hunk 2 above)
Comment (StephenWeatherford) [reply quickstarts/microsoft.databricks/databricks-workspace/main.bicep:35]: @msaranga Thanks for reviewing.  No, the 'existing' keyword indicates it's just saying it already exists (and they we can get its .id from it).

**Comment by azure-quickstarts** [comment]
@mamccrea - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Pull request merged**